include/libopencm3/stm32/f4/rcc.h

```diff
@@ -115,6 +115,10 @@
 /* PLLM: [5:0] */
 #define RCC_PLLCFGR_PLLM_SHIFT			0
 
+#define RCC_PLLCFGR_NO_R_RESERVED 0xf0bc8000
+#define RCC_PLLCFGR_R_RESERVED    0x80bc8000
```

Comment:
even without my earlier comments on these, (and the same applies to the new ones you're adding here) moving these defines is out of scope of this commit.